default-config.json

```diff
@@ -50,20 +50,30 @@
       "alias": "Cadence play "
     }
   ],
+  "commandFriendlyNames": {
```

Comment:
Should we merge this with `commandDescriptions`?

ie

```json
"commandDescriptions": {
    "play": {
      "description": "Starts playing Cadence in your voice channel", 
      "friendly": "play" 
    },
    ... 
} 
```

Reply:
Oh yes, good idea

Reply:
I suppose if we do that it should be clear and say friendlyName or something though, shouldn't it...

Reply:
Yeah, friendlyName is probably best. I'll make it that.